src/styles/typography/label.tsx

```diff
@@ -23,6 +24,7 @@ export const LargeLabel = ({ children, weight }: LargeLabelProps) => {
         fontSize: typography.size.large,
         fontWeight: fontWeight,
         color: colors.PrimaryXdark,
+        ...style,
```

Comment:
como o style pode ser nullable (vc colocou o `?` la na interface), esse destruct pode quebrar sua aplicacao para casos de `undefined`. Sendo assim, é mais seguro fazer isso:

```suggestion
        ...(style ?? {}),
```